mod-audit-server/src/main/java/org/folio/rest/impl/InitAPIs.java

```diff
@@ -64,27 +79,26 @@ private Future<?> deployConsumersVerticles(Vertx vertx) {
     Promise<String> orderEventsConsumer = Promise.promise();
     Promise<String> orderLineEventsConsumer = Promise.promise();
     Promise<String> pieceEventsConsumer = Promise.promise();
+    Promise<String> invoiceEventsConsumer = Promise.promise();
 
-    vertx.deployVerticle(getVerticleName(verticleFactory, OrderEventConsumersVerticle.class),
-      new DeploymentOptions()
-        .setWorker(true)
-        .setInstances(acqOrderConsumerInstancesNumber), orderEventsConsumer);
-
-    vertx.deployVerticle(getVerticleName(verticleFactory, OrderLineEventConsumersVerticle.class),
-      new DeploymentOptions()
-        .setWorker(true)
-        .setInstances(acqOrderLineConsumerInstancesNumber), orderLineEventsConsumer);
-
-    vertx.deployVerticle(getVerticleName(verticleFactory, PieceEventConsumersVerticle.class),
-      new DeploymentOptions()
-        .setWorker(true)
-        .setInstances(acqPieceConsumerInstancesNumber), pieceEventsConsumer);
+    deployVerticle(vertx, verticleFactory, OrderEventConsumersVerticle.class, acqOrderConsumerInstancesNumber, acqOrderConsumerPoolSize, orderEventsConsumer);
```

Comment:
Similar to what we did in mod-orders-storage: 

- <https://github.com/folio-org/mod-orders-storage/blob/14b873de9dcc3fdbe40fabe5920a6961e6a353c2/src/main/java/org/folio/rest/impl/InitAPIs.java#L115>